tests/api/exercises/GetCest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace BusuuTest\Tests;
+
+use ApiTester;
+use Codeception\Util\HttpCode;
+
+class GetCest
+{
+    public function testGetExercises_ExercisesExist_ReturnAllExercises(ApiTester $I)
+    {
+        $I->wantToTest('valid request to get all exercises');
+        $I->sendGET('/exercises');
+        $I->seeResponseCodeIs(HttpCode::OK);
+        $I->seeResponseIsJson();
+        $I->seeResponseMatchesJsonType([
+            'exerciseId' => 'integer',
+            'exerciseAuthor' => 'string',
+            'exerciseText' => 'string'
+        ]);
+    }
+
+    public function testGetExercise_InvalidParam_ReturnError(ApiTester $I)
+    {
+        $I->wantToTest('invalid request param to get specific exercise');
+        $I->sendGET('/exercises/1shirt');
+        $I->seeResponseCodeIs(HttpCode::BAD_REQUEST);
```

Comment:
You could (in the future) refactor testing the failures with your own Codeception Helper method (something like `seeResponseIsBusuuFailure()` that does all the needed check